Library/Formula/abcde.rb

```diff
@@ -22,6 +23,11 @@ class Abcde < Formula
   depends_on "vorbis-tools" => :optional
   depends_on "glyr" => :optional
 
+  patch do
+    url "http://abcde.einval.com/bugzilla/attachment.cgi?id=12"
+    sha256 "4620dd5ef7ab32b6511782da85831661ccee292fadff1f58acc3b4992486af62"
+  end
```

Comment:
This patch comes from upstream and is already included there. However, no new release yet, thus the "downporting". Using --HEAD didn't work for me either, see issue #47643.
I'll add the comment.